### PR TITLE
#3156 Colour -> Color

### LINKED ISSUE #3156: Replace spelling of colour -> color
## Describe the bug

In a few places, `colour` is used over `color`

### To reproduce

Internal code issue

### Expected behaviour

Should be `color` not `colour`

### Proposed Solution

Find and replace all instances of `colour` with `color`

### Version and device info

- App version: 6.0
- Tablet model: N/A
- OS version: N/A

### Additional context

N/A

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #3156

## Change summary

- Changes some spelling in the app from colour -> color
- Unfortuantely named a field in `TemperatureBreachConfiguration` colour - leaving as is

## Testing

- [ ] Internal code change, not much testing to do?

### Related areas to think about

N/A

### CODE REVIEW

File: src/widgets/FormControl.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { connect } from 'react-redux';
 import { PageButton } from './PageButton';
 
 import { FORM_INPUT_TYPES } from '../utilities/formInputConfigs';
-import { FormTextInput } from './FormInputs/FromTextInput';
+import { FormTextInput } from './FormInputs/FormTextInput';
```
Comment (joshxg) [inline src/widgets/FormControl.js:12]: Renamed this file :(

Comment (Chris-Petty) [reply src/widgets/FormControl.js:12]: Sneaky when the misspelling is a valid spelling heh

**Review by Chris-Petty — APPROVED** [review]
Thanks for satisfying my pedantry on this one. The vast majority of references in mobile were already using `color`. Even in this diff there were spots where the same line had both spellings, so less likely to have people subtly pass undefined things about.

**Comment by Chris-Petty** [comment]
As you've highlighted could potentially make the getter consistent, but I'm on side with not conflicting with the data source in this case!
image

**Comment by joshxg** [comment]
All good. Just confusion from me - people complaining about american spelling and making you change names etc!

**Pull request merged**